apps/server/src/modules/board/service/board-do-copy-service/swap-internal-links.visitor.ts

```diff
@@ -0,0 +1,60 @@
+import {
+	AnyBoardDo,
+	BoardCompositeVisitor,
+	Card,
+	Column,
+	ColumnBoard,
+	LinkElement,
+	SubmissionContainerElement,
+	SubmissionItem,
+} from '@shared/domain/domainobject';
+import { DrawingElement } from '@shared/domain/domainobject/board/drawing-element.do';
+import { EntityId } from '@shared/domain/types';
+
+export class SwapInternalLinksVisitor implements BoardCompositeVisitor {
+	constructor(map: Map<EntityId, EntityId>) {
```

Comment:
Can also just be `constructor(private (readonly) map: Map<EntityId, EntityId>)` with the instance variable declaration removed. Is probably a matter of taste, just saying... 🤓